packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -338,6 +356,29 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
                         </Col>
                     </Row>
                 )}
+
+                {allowTimepointProperties && !appPropertiesOnly && (
+                    <Row className="margin-top">
+                        <Col xs={2}> Auto-Link Data to Study </Col>
+                        <Col xs={5}>
+                            <FormControl
+                                componentClass="select"
+                                id={ENTITY_FORM_IDS.AUTO_LINK_TARGET}
+                                onChange={this.onFormChange}
+                                value={model.autoLinkTargetContainerId || ''}
+                            >
+                                <option key="_empty" value={null} />
+                                {containers &&
+                                    containers.map((container, i) => (
```

Comment:
Do you want to match the `AssayPropertiesInput` component case for the loading state of this? I also wonder if it might be worth factoring out this usage and the other one into a shared `AutoLinkToStudySelect` component (name TBD) since they are so similar.

Reply:
Good point! Factored out into `AutoLinkToStudyDropdown`